packages/apps/docs/src/pages/docs/pact/index.tsx

```diff
@@ -0,0 +1,151 @@
+                </Stack>
+              </Stack>
+            </Card>
+          </Grid.Item>
+          {popularPages.length > 0 && (
+            <Grid.Item
+              columnSpan={{
+                sm: 10,
+                md: 3,
+              }}
+            >
+              <MostPopular pages={popularPages} title="Most viewed pact docs" />
+            </Grid.Item>
+          )}
+        </Grid.Root>
+      </Stack>
+      <Stack direction="column" spacing="$md">
+        <Heading as="h6">Browse by Resources</Heading>
+        <Stack direction="column" spacing="$md">
+          <BrowseSection title="General" titleAs="h6" direction="row">
+            <BrowseSection.LinkBlock
+              title="Overview of Pact"
+              subtitle="Explore all products"
+              icon="Overview"
+              href="/docs/pact"
+            />
+            <BrowseSection.LinkBlock
+              title="Chainweb"
+              subtitle="Explore all products"
+              icon="SmartContract"
+              href="/docs/chainweb"
+            />
+            <BrowseSection.LinkBlock
+              title="Marmalade"
+              subtitle="Explore all products"
+              icon="Marmalade"
+              href="/docs/marmalade"
+            />
+          </BrowseSection>
+          <BrowseSection title="Pact" titleAs="h6" direction="row">
+            <BrowseSection.LinkBlock
+              title="Pact Language"
+              subtitle="Explore all products"
+              icon="PactLanguage"
+              href="/docs/pact"
+            />
+            <BrowseSection.LinkBlock
+              title="Useful Tools"
+              subtitle="Explore all products"
+              icon="UsefulTools"
+              href="/docs/build/tools"
+            />
+            <BrowseSection.LinkBlock
+              title="Beginner Tutorials"
+              subtitle="For starters"
+              icon="PactDeveloper"
+              href="/docs/pact/beginner"
+            />
+            <BrowseSection.LinkBlock
+              title="Intermediate Tutorials"
+              subtitle="get some more experience"
+              icon="PactDeveloper"
+              href="docs/pact/intermediate"
+            />
+            <BrowseSection.LinkBlock
+              title="Quickstart"
+              subtitle="Explore all products"
+              icon="QuickStart"
+              href="/docs/chainweb"
+            />
+          </BrowseSection>
+        </Stack>
+      </Stack>
+    </Stack>
+  );
+};
+
+export const getStaticProps: GetStaticProps = async () => {
+  const mostPopularPages = await getMostPopularPages('/docs/pact');
+  return {
+    props: {
+      popularPages: mostPopularPages,
+      leftMenuTree: checkSubTreeForActive(getPathName(__filename)),
+      frontmatter: {
+        title: 'Pact',
+        menu: 'Pact',
+        subTitle: 'The safest, most user-friendly language for smart contracts',
+        label: 'Introduction',
+        order: 0,
```

Comment:
order should be `2`.
the order of the index page is the order in the parentTree.
in its own tree its order will always be 0